app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -327,6 +342,17 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         toolbarView.view.requestFocus()
     }
 
+    private fun hideDeviceKeyboard() {
+        val imm =
+            requireContext().getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+        imm.toggleSoftInput(InputMethodManager.HIDE_IMPLICIT_ONLY, 0)
+    }
```

Comment:
Can we use Fragment.hideKeyboard() instead?

Reply:
No we need these flags specifically but I can add a comment to explain